design/velero-uploader-configuration.md

```diff
@@ -122,5 +122,44 @@ Roughly, the process is as follows:
 3. Each respective controller within the CRs calls the uploader, and the ParallelFilesUpload from UploaderConfig in CRs is passed to the uploader. 
 4. When the uploader subsequently calls the Kopia API, it can use the ParallelFilesUpload to set the MaxParallelFileReads parameter, and if the uploader calls the Restic command it would output one warning log for Restic does not support this feature.
 
+### Sparse Option For Kopia & Restic Restore
+In many system files, there are numerous zero bytes or empty blocks that still occupy physical storage space. Sparse backup employs a more intelligent approach by only backing up the actual data-containing portions. For those empty blocks or zero bytes, it merely records their presence without actually storing them. This can significantly reduce the storage space required for backups, especially in situations where there is a substantial amount of empty data in large file systems.
+
+Below are the key steps that should be added to support this new feature.
+#### Velero CLI 
+The Velero CLI will support a `--write-sparse-files` flag, allowing users to set the `WriteSparseFiles` value when creating restores with Restic or Kopia uploader.
```

Comment:
Can we just call the option "--sparse"?

Reply:
thanks for your comment. 

Here, I just want to align the flag with Kopia